src/cloudant/client.py

```diff
@@ -788,9 +788,17 @@ def bluemix(cls, vcap_services, instance_name=None, service_name=None, **kwargs)
             print client.all_dbs()
         """
         service_name = service_name or 'cloudantNoSQLDB'  # default service
-        service = CloudFoundryService(vcap_services,
-                                      instance_name=instance_name,
-                                      service_name=service_name)
+        try:
+            service = CloudFoundryService(vcap_services,
+                                          instance_name=instance_name,
+                                          service_name=service_name)
+        except CloudantException:
```

Comment:
Do we need this try/except? Why not just let it throw?

Reply:
I wanted to keep the exceptions consistent with each class exception.  Since `CloudFoundryService` is used during client creation, we could consider updating all `CloudantException` in the function to `CloudantClientException`.